PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,29 @@
-Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.
+<!--
+    Thanks for opening a Pull Request (PR). Please read through
+    these instructions to help us review and merge your change quicker.
 
-Please delete anything that isn't relevant to your patch.
+    Below, replace Fixes #ISSUE_NUMBER with the issue related to what
```

Comment:
It might be even clearer if we just said: "replace ISSUE_NUMBER with the number of the issue related to what..."

Reply:
Good idea